Directory.Build.props

```diff
@@ -11,7 +11,7 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <HighEntropyVA>true</HighEntropyVA>
-    <IdentityModelTokenPackageVersion>7.0.3</IdentityModelTokenPackageVersion>
+    <IdentityModelTokenPackageVersion>6.34.0</IdentityModelTokenPackageVersion>
```

Comment:
This version is unlisted in NuGet.org. Are you sure we want this version?

Reply:
Using the previous released version.